File: Project/workspace/MapYou/src/it/mapyou/model/MapmeState.java
/**
 * 
 */
package it.mapyou.model;

import android.os.Parcelable;

/**
 * @author mapyou (mapyouu@gmail.com)
 *
 */
public interface MapmeState extends Parcelable{

	public static final String ON_CREATE = "on_create";

	public static final String ON_DESTROY = "on_destroy";

	public static final String ON_START = "on_start";
	
	public String getState();
}

File: Project/server/mapyou_server/core/MapmeState.php
<?php
class MapmeState{
	static function on_create_mapme (){
		return "on_create";
	}

	static function on_start_mapme (){
		return "on_start";
	}
	
	static function on_destroy_mapme (){
		return "on_destroy";
	}
	
}
?>